src/libraries/System.Collections/src/System/Collections/Generic/CollectionExtensions.cs

```diff
@@ -9,13 +9,13 @@ namespace System.Collections.Generic
     public static class CollectionExtensions
     {
         [return: MaybeNull]
-        public static TValue GetValueOrDefault<TKey, TValue>(this IReadOnlyDictionary<TKey, TValue> dictionary, TKey key) where TKey : notnull
+        public static TValue GetValueOrDefault<TKey, TValue>(this IReadOnlyDictionary<TKey, TValue> dictionary, TKey key)
         {
             return dictionary.GetValueOrDefault(key, default);
         }
 
         [return: MaybeNull]
```

Comment:
Nice. This needs to be `MaybeNullIfNull("defaultValue")`, but that doesn't exist.